app/src/main/res/layout/admin_pin_activity.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:profile="http://schemas.android.com/tools">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profile.AdminPinViewModel" />
+  </data>
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/admin_auth_container"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/addProfileBackground">
+      <TextView
+        android:id="@+id/admin_pin_main_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/admin_pin_requirement_description"
+        android:textColor="@color/oppiaPrimaryText"
+        android:textSize="16sp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginTop="32dp"
+        android:paddingStart="32dp"
+        android:paddingEnd="32dp"/>
+      <TextView
+        android:id="@+id/admin_pin_warning_text"
```

Comment:
Shouldn't this be visibility gone/invisible by default?

Reply:
No this text should be shown. I didn't use the android:text attribute because the word "not" has to be bold, so I set the text in the presenter.
`binding.adminPinWarningText.text = activity.getText(R.string.admin_pin_warning)`

Reply:
Ah what I mean is, shouldn't this be invisible unless there is an error? Otherwise we're telling Android to render something (even if that something doesn't have any text set).

Reply:
No this isn't error text (I'll rename it). It is additional text that warns the user not to use a pin associated with an existing account.